src/ui/components/AccountOverview.js

```diff
@@ -9,10 +9,15 @@ import TableHead from "@material-ui/core/TableHead";
 import TableRow from "@material-ui/core/TableRow";
 import Paper from "@material-ui/core/Paper";
 import {type Account} from "../../ledger/ledger";
+import {type CurrencyDetails} from "../../api/currencyConfig";
 import * as G from "../../ledger/grain";
 import {useLedger} from "../utils/LedgerContext";
 
-export const GrainAccountOverview = () => {
+type OverviewProps = {|currency: CurrencyDetails|};
```

Comment:
```suggestion
type OverviewProps = {|+currency: CurrencyDetails|};
```
Should be readonly, right?